examples/isentropic_example.py

```diff
@@ -0,0 +1,227 @@
+
+########################################
+# A quick look at the shape of these variables will show that the data is now in isentropic
+# coordinates, with the number of vertical levels as specified above.
+
+print(isentprs.shape)
+print(isentspech.shape)
+print(isentu.shape)
+print(isentv.shape)
+print(isenttmp.shape)
+print(isenthgt.shape)
+
+#################################
+# **Converting to Relative Humidity**
+#
+# The NARR only gives specific humidity on isobaric vertical levels, so relative humidity will
+# have to be calculated after the interpolation to isentropic space.
+
+isentrh = mcalc.relative_humidity_from_specific_humidity(isentspech, isenttmp, isentprs)
+
+#######################################
+# **Plotting the Isentropic Analysis**
+
+
+# Set up our projection
+crs = ccrs.LambertConformal(central_longitude=-100.0, central_latitude=45.0)
+
+
+# Set up our array of latitude and longitude values and transform to
+# the desired projection.
+tlatlons = crs.transform_points(ccrs.PlateCarree(), lon, lat)
+tlons = tlatlons[:, :, 0]
+tlats = tlatlons[:, :, 1]
+
+# Limit the extent of the map area, must convert to proper coords.
+LL = crs.transform_point(-122., 22., ccrs.PlateCarree())
+UR = crs.transform_point(-60., 47., ccrs.PlateCarree())
+
+# Choose a level to plot, in this case 296 K
+level = 0
+
+# Get data to plot state and province boundaries
+states_provinces = cfeature.NaturalEarthFeature(category='cultural',
+                                                name='admin_1_states_provinces_lakes',
+                                                scale='50m',
+                                                facecolor='none')
+
+fig = plt.figure(1, figsize=(17., 12.))
+ax = plt.subplot(111, projection=crs)
+#   ax.set_extent([west long, east long, south lat, north lat])
+ax.set_extent([LL[0], UR[0], LL[1], UR[1]], crs)
+ax.coastlines('50m', edgecolor='black', linewidth=0.75)
+ax.add_feature(states_provinces, edgecolor='black', linewidth=0.5)
+
+# Plot the surface
+clevisent = np.arange(0, 1000, 25)
+cs = ax.contour(tlons, tlats, isentprs[level, :, :], clevisent,
+                colors='k', linewidths=1.0, linestyles='solid')
+plt.clabel(cs, fontsize=10, inline=1, inline_spacing=7,
+           fmt='%i', rightside_up=True, use_clabeltext=True)
+
+# Plot RH
+cf = ax.contourf(tlons, tlats, isentrh[level, :, :], range(10, 106, 5),
+                 cmap=plt.cm.gist_earth_r)
+cb = plt.colorbar(cf, orientation='horizontal', extend=max, aspect=65, shrink=0.5, pad=0,
+                  extendrect='True')
+cb.set_label('Relative Humidity', size='x-large')
+
+# Transform Vectors before plotting, then plot wind barbs.
+ut, vt = crs.transform_vectors(ccrs.PlateCarree(), lon, lat, isentu[level, :, :].m,
+                               isentv[level, :, :].m)
+ax.barbs(tlons, tlats, ut, vt, length=6, regrid_shape=20)
+
+# Make some titles
+plt.title('%3d K Isentropic Pressure (hPa), Wind (kt), Relative Humidity (percent)'
```

Comment:
Flake8 is complaining about you using `%` for string formatting rather than `.format()`. So this should be:
```python
plt.title('{:3d} K Montgomery Streamfunction ($10^{-2}m^{2}s^{-2}$), Wind (kt), '
          'Relative Humidity (percent)'.format(isentlevs[level].m), loc='left')
```